src/components/modal/ModalBackdropConnected.tsx

```diff
@@ -0,0 +1,25 @@
+import { IReactVaporState, IReduxActionsPayload } from '../../ReactVapor';
+import { IModalBackdropOwnProps, IModalBackdropStateProps, IModalBackdropProps, IModalBackdropDispatchProps, ModalBackdrop } from './ModalBackdrop';
+import { IReduxAction, ReduxUtils } from '../../utils/ReduxUtils';
+import { connect } from 'react-redux';
+import * as _ from 'underscore';
+
+
+const mapStateToProps = (state: IReactVaporState, ownProps: IModalBackdropOwnProps): IModalBackdropStateProps => {
+  let modals = state.modals;
+  if (ownProps.displayFor && ownProps.displayFor.length > 0) {
+    modals = modals.filter(modal => {
+      return _.contains(ownProps.displayFor, modal.id);
+    });
```

Comment:
You can inline the function (if it's under 140 chars)

`modals = modals.filter(modal => _.contains(ownProps.displayFor, modal.id));`